databases/week2/db1w2_schoolDB_DDL-DML_hwPart2.sql

```diff
@@ -8,55 +8,43 @@ CREATE TABLE `Class` (
   `begin_date` DATETIME NOT NULL,
   `end_date` DATETIME NULL
 ) ENGINE = InnoDB DEFAULT CHARSET = utf8mb4 COLLATE = utf8mb4_unicode_ci;
+-- renaming the columns: begin_date to begins and end_date to ends
+ALTER TABLE
+  Class RENAME COLUMN begin_date TO begins;
+ALTER TABLE
+  Class RENAME COLUMN end_date TO ends;
 -- creating Student table : with the columns: id, name, email, phone, class_id (foreign key)
-CREATE TABLE `Student` (
-  `id` INT(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
-  `name` VARCHAR(255) NOT NULL,
-  `email` VARCHAR(255) NOT NULL,
-  `phone` VARCHAR(255) NULL,
-  `class_id` INT(10) UNSIGNED,
-  FOREIGN KEY (`class_id`) REFERENCES `Class` (`id`) ON DELETE CASCADE ON UPDATE CASCADE
-) ENGINE = InnoDB DEFAULT CHARSET = utf8mb4 COLLATE = utf8mb4_unicode_ci;
+  CREATE TABLE `Student` (
+    `id` INT(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    `name` VARCHAR(255) NOT NULL,
+    `email` VARCHAR(255) NOT NULL,
+    `phone` VARCHAR(255) NULL,
+    `class_id` INT(10) UNSIGNED,
+    FOREIGN KEY (`class_id`) REFERENCES `Class` (`id`) ON DELETE CASCADE ON UPDATE CASCADE
+  ) ENGINE = InnoDB DEFAULT CHARSET = utf8mb4 COLLATE = utf8mb4_unicode_ci;
 -- Add a new column to the Class table named status which can only have the following VALUES: not-started, ongoing, finished (hint: enumerations).
+  -- *adding column using enum
+  -- ALTER TABLE Class ADD FULLTEXT INDEX idx_school_student_name (name);
 ALTER TABLE
   Class
 ADD
-  status VARCHAR(255);
+  status1 enum('not-started', 'on-going', 'finished') NOT NULL;
 ALTER TABLE
-  Class
-ADD
-  ClassRemarks VARCHAR(255);
--- delete column from the table
+  Class RENAME COLUMN status1 TO status;
+-- ALTER TABLE Class ADD status VARCHAR(255);
+  -- delete column from the table
 ALTER table
-  Class DROP COLUMN ClassRemarks;
+  Class DROP COLUMN status;
```

Comment:
you are dropping the Status column which you added in a few line on above, are you sure you want to drop it?!